bindings/muen/muen-net.c

```diff
@@ -138,27 +185,39 @@ void net_init(void)
 
     if (!chan_in) {
         log(WARN, "Solo5: Net: No input channel\n");
```

Comment:
I suggest modifying this and other messages in this function to include the manifest device name, otherwise it'll be impossible to diagnose in the presence of multiple devices.